src/sim.rs

```diff
@@ -333,11 +335,21 @@ impl<'a> Sim<'a> {
         f(top.iter_mut())
     }
 
-    /// Run the simulation to completion.
+    /// Run the simulation until all client hosts have completed.
```

Comment:
Is there a standard way to keep the simulation alive? A client that just spins?

Could add a follow up comment along the lines of how to keep the sim running if it makes sense.

Reply:
I think a sim with no clients is pretty niche. You'd essentially be running your sim and checking for something to err or panic. This was originally designed to have one or more hosts and then a client that drives the sim, or tests it. Registering a client that just polls forever would work, but I wonder if it'd be better to provide a different execution method instead, as an alternative to run().